data/korath/korath missions.txt

```diff
@@ -257,3 +257,208 @@ mission "Small Scale Delivery"
+			`	He leads you into the ship, shouting into a communication device that is then picked up on speakers throughout the ship. You can only glance around as you move through the engineering section but it's clear their System Core was destroyed and left them stranded.`
+			`	The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree with more than half incapacitated or unconscious. Even if if the ship is spaceworthy, there aren't enough crew to fly it.`
+			`	You quickly count the number of surviors and communicate to your guide that you can take them all. He mimics your nodding head gesture and shows you his plams and makes quick cirlcles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`
+	
+	on complete
+		payment 3000000
+		conversation
+			`The Efreti commander hails the authorities once you enter the system and they have medical crews waiting when you land. It takes several hours to evacute the injured as doctors or maybe paramedics swarm over them to triage, treat, and prepare for transport. The three who died during the flight are treated with respect and carried off the <ship>.`
+			`	When you make it to the spaceport you find a message from the Quarg. "Bold Human! With profuse thanks we offer you for helping the stranded Korath when we could not. The Korath ever mourn the loss of their worlds and from time to time brave the dangers to seek what they can salvage among the ruins and to seek out those where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
+			`	Your communicator notifies you of a payment of <payment>.`
+
+
+mission "Efreti: Distress Call 2"
+	name "Recover the fallen"
+	description "Recover the bodies of the dead Korath on <stopovers> and return them to <destination>."
+	source "Laki Nemparu"
+	stopover "Seleptra Nak"
+	destination "Karek Fornati"
+	landing
+
+	to offer
+		has "Efreti: Distress Call 1: done"
+
+	on offer
+		conversation
+			`A delegation of Korath approach you somewhat hesitantly in the spaceport. They are dressed in what appears to be much finer clothing than most of the Korath you have met, certainly nothing you would wear to do work in a spaceport. As they make the traditional gesture of greeting, which you return, you are curious both as to what they want and how they will ask.`
```

Comment:
This sounds like a little time has passed from the previous mission. Right now, this will immediately pop up after completing the conversation of the previous mission. There should probably be an event to spread them out, even if just for a few days.